deployments/examples/placements/README.md

```diff
@@ -20,13 +20,22 @@
 App placements rules in Yunikorn contains `Provided Rule`, `User Name Rule`, `Fixed Rule`, `Tag Rule`.
 Every placement example includes a example yaml file and a config yaml file.
 The sleep pod is described in the example file and the partition is provided in the config yaml file.
-Before deploying the pods, the configuration field in yunikorn-release/helm/yunikorn/value.yaml should be replaced by the configuration field in the config.yaml.
 
 * [Provided Rule](./provided)
 * [User Name Rule](./username)
 * [Fixed Rule](./fixed)
 * [Tag Rule](./tag)
 
+## How to adopt the queues in `config.yaml`
+Before deploying the pods, threre are some modifications required in [release/value.yaml](yunikorn-release/helm/yunikorn/value.yaml).
+1. Replace the configuration field by the configuration in the `config.yaml`.
+2. Add `queue.yaml: ""` in `yunikornDefaults` field.
+```
+yunikornDefaults: {
+  queue.yaml: ""
```

Comment:
This should be `queues.yaml`, not `queue.yaml`. Also, why are we setting this to an empty string and not simply to the new configuration?

Reply:
I trace the code in the https://github.com/apache/yunikorn-release/blob/master/helm-charts/yunikorn/templates/yunikorn-defaults.yaml#L60-L68。
It seems to get the key but it will not get the value.
So I use empty string here.
Is there a suggested value of the queues.yaml?